Team9ProjectTests/src/com/indragie/comput301as1/test/ReceiptTests.java

```diff
@@ -0,0 +1,17 @@
+package com.indragie.comput301as1.test;
+
+import com.indragie.cmput301as1.ExpenseItemAddActivity;
+
+import android.test.ActivityInstrumentationTestCase2;
+
+public class ReceiptTests extends ActivityInstrumentationTestCase2<ExpenseItemAddActivity> {
+
+	public ReceiptTests(String name) {
```

Comment:
My bad, I did have some tests in the works but I guess I forgot to update them here. Also, I couldn't figure out how to test scaledImage, i.e. a .jpeg. I'm not sure how to simulate taking/creating a .jpeg file and then save it somewhere temporary to test it. Does anyone have any advice?